docs/modules/ROOT/pages/references/crds.adoc

```diff
@@ -732,8 +732,8 @@ VSHNPostgreSQL is the API for creating Postgresql clusters.
 [cols="25a,75a", options="header"]
 |===
 | Field | Description
-| *`schedule`* __string__ | 
-| *`retention`* __integer__ | 
+| *`schedule`* __string__ | Schedule how often should backup be made,example: '30 23 * * *' - each day at 23:30. Please note that this isn't a regular Cron syntax.
```

Comment:
Here you write "this isn't a regular Cron syntax" and further down "standard regex: https://en.wikipedia.org/wiki/Cron". What's true now?